lib/core/domain/onboarding/onboarding_inputs.dart

```diff
@@ -17,7 +17,7 @@ class GetProfileInput with _$GetProfileInput {
 class UpdateUserProfileInput with _$UpdateUserProfileInput {
   factory UpdateUserProfileInput({
     required String username,
-    List<String>? uploadPhoto,
+    @JsonKey(name: 'new_photos') List<String>? uploadPhoto,
```

Comment:
should be same name with backed in camelCase format 
ex: newPhotos

Reply:
Backend is requiring this in snake format

Reply:
What I mean here is 
`@JsonKey(name: 'new_photos') List<String>? newPhotos` 
instead of 
@JsonKey(name: 'new_photos') List<String>? `uploadPhoto`

Reply:
The purpose of putting a jsonkey here is to separate variable name from backend required and local name on app. Using `newPhotos` name make no sense what is the purpose of this variable. Make it hard to debugging in the future.

Reply:
I dont think newsPhoto makes no sense here, in the context of updating profile, newPhotos good enough to understand

Reply:
Feel good and agreed with Ron, I think we can keep like this too 👍

Reply:
👍